### DISCUSSION TITLE
Will Ghelper reduce voltage override bios settings?

### DISCUSSION THREAD

**Discussion by Sparthon** [seerge/g-helper#3308] [Q&A] [answered]
If I want to use the bios settings, how should Ghelper be set, because I set the Ghelper settings and it doesn't feel effective
![542b052d-02e8-4881-a9c1-dcf84869edb5](https://github.com/user-attachments/assets/91ca56ba-900a-4db0-a289-505217306791)

**Comment by seerge** [comment, marked as answer]
@Sparthon if you want to use defaults, you don't need to apply any custom undervolting.

**Reply by Sparthon** [reply]
OK,thanks
